README.md

```diff
@@ -26,20 +26,39 @@
 
 We’ve spent thousands of hours grappling with traditional CRMs like Pipedrive and Salesforce to align them with our business needs, only to end up frustrated — customizations are complex and the closed ecosystems of these platforms can feel restrictive.
 
-We felt the need for a CRM platform that empowers rather than constrains. We believe the next great CRM will come from the open source community. And we’ve packed Twenty with powerful features to give you full control and help you run your business efficiently. 
+We felt the need for a CRM platform that empowers rather than constrains. We believe the next great CRM will come from the open source community. And we’ve packed Twenty with powerful features to give you full control and help you run your business efficiently.
 
 <br>
 
-# Demo 
+# Demo
 Go to <a href="https://demo.twenty.com/">demo.twenty.com</a> and login with the following credentials:
 ```
 email: noah@demo.dev
 password: Applecar2025
 ```
 
-See also:  
-🚀 [Self-hosting](https://docs.twenty.com/start/self-hosting/)  
-🖥️ [Local Setup](https://docs.twenty.com/start/local-setup)  
+# Installation Guide
```

Comment:
I think this can be a bit confusing to add in the README because "installation" is local setup for many.
Let's just add it to self-hosting/docker-compose.mdx as a v1 and then once we have a little more polished experience we update the README?